packages/ui/components/form-core/src/validate/Validator.js

```diff
@@ -32,6 +32,7 @@ export class Validator extends EventTarget {
      * @type {ValidationType}
      */
     this.type = config?.type || 'error';
+    this._$isValidator$ = true;
```

Comment:
@tlouisse, better to do `this['_$isValidator$']` instead?

Reply:
Yes indeed, because otherwise it can get renamed and two different lion bundles will end up with different names